### PR TITLE
feat: Standalone snippets always import own namespace.

### PR DESCRIPTION
**Pull request by amanda-tarafa**

### CODE REVIEW

**Comment by amanda-tarafa** [comment]
First commit is the production code change.
Second commit is the change in the expected output for tests.

File: Google.Api.Generator/CodeGenerator.cs [hunk 1]
```diff
@@ -138,7 +138,7 @@ internal static class CodeGenerator
                     // Generate snippets for the service
                     // TODO: Consider removing this once we have integrated the standalone snippets
                     // with docs generation.
-                    var snippetCtx = SourceFileContext.CreateUnaliased(clock, s_wellknownNamespaceAliases, s_avoidAliasingNamespaceRegex);
+                    var snippetCtx = SourceFileContext.CreateUnaliased(clock, s_wellknownNamespaceAliases, s_avoidAliasingNamespaceRegex, true);
```
Comment (jskeet) [inline Google.Api.Generator/CodeGenerator.cs:141]: It might be worth using named arguments both here and for the other call, just for clarity.

**Review by jskeet — APPROVED** [review]
One style-only suggested change, but otherwise looks good.

File: Google.Api.Generator/CodeGenerator.cs (hunk 1 above)
Comment (amanda-tarafa) [reply Google.Api.Generator/CodeGenerator.cs:141]: Yes, done.

**Review by amanda-tarafa — COMMENTED** [review]
Comment addressed. I'll squash the second commit and wait for autosynth to be done before merging.

**Pull request merged**